### PR TITLE
Q2 s4 gitops environments

### PR DESCRIPTION
**Pull request by NimRegev**

### CODE REVIEW

File: _docs/dashboards/gitops-environments.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+
+1. In the Codefresh UI, from the Ops in the sidebar, select **Environments**, and then click **Add Environment**.
+1. Define the following:
+    1. **Name**: A unique name for your environment, which is meaningful in the context of your development and deployment cycle. 
+    1. **Kind**: The purpose of this environment, and can be either **Production** where the live versions of the applications are deployed,  or **Non-production** where development, testing, staging versions are deployed.
+    1. **Tags**: Any metadata providing additional context and information about the environment, used for filtering and organization purposes.
+    1. **Clusters and Namespaces**: Single or multiple cluster-namespace pairs to map to the environment. Adding a cluster with one or more namespaces populates the environment with all the applications deployed in the namespaces. When selecting namespaces in a cluster, use `*` as a wildcard for pattern-based matching. For example, you can use `prod-*` to add all namespaces with names starting with `prod-`.
```
Comment (denis-codefresh) [inline _docs/dashboards/gitops-environments.md:74]: Wildcarding is not supported yet and we don't have it in plan. Better to not document it for now.

File: _docs/dashboards/gitops-environments.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+### Trace applications across environments
+Trace the same application as it moves across different environments in its development, testing, and deployment cycle. See the version of the application running in each environment, the most recent commit indicating the change, and the user who made the commit. 
+
+
+Alternatively, track a set of applications deployed to multiple environments of the same kind and at the same level. For example, track the billing application deployed to multiple production environments based on regions.
+Here too, see which applications are running on each environment, the most recent commit to the application, and the user who made the commit. 
+
+
+1. In the Codefresh UI, from the Ops in the sidebar, select **Environments**.
+1. Switch to **Detailed** view.
```
Comment (denis-codefresh) [inline _docs/dashboards/gitops-environments.md:154]: Incorrect screenshot below

File: _docs/dashboards/gitops-environments.md (hunk 1 above)
Comment (NimRegev) [reply _docs/dashboards/gitops-environments.md:74]: Done

File: _docs/dashboards/gitops-environments.md (hunk 2 above)
Comment (NimRegev) [reply _docs/dashboards/gitops-environments.md:154]: fixed

**Pull request merged**